migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```

Comment:
This is normally used for older Python versions to import something from future versions. Are you sure we need this, considering we want to use the latest Python version?